main.py

```diff
@@ -52,15 +58,32 @@ def main():
         if config.get("schedule"):
             yin_yang.start_daemon()
         else:
-            print("looks like you did not specified a time")
-            print("You can use the gui with yin-yang -gui")
-            print("Or edit the config found in ~/.config/yin_yang/yin_yang.json")
-            print("You need to set schedule to True and edit the time to toggles")
+            logger.warning(
+                "Looks like you have not specified a time."
+                "You can use the GUI by running Yin & Yang or "
+                "edit the config found in ~/.config/yin_yang/yin_yang.json."
+                "You need to set schedule to true and edit the time to toggles."
+            )
```

Comment:
I feel like this is not useful in the logs. Lets keep the print statement here since this is user facing info. In future we could point to the wiki on how to set up yin-yang without the gui.